Makefile

```diff
@@ -450,7 +441,7 @@ compilepy3:
 .st2common-circular-dependencies-check:
 	@echo "Checking st2common for circular dependencies"
 	find ${ROOT_DIR}/st2common/st2common/ -name \*.py -type f -print0 | xargs -0 cat | grep st2reactor ; test $$? -eq 1
-	find ${ROOT_DIR}/st2common/st2common/ \( -name \*.py ! -name runnersregistrar\.py -name \*.py ! -name compat\.py | -name inquiry\.py \) -type f -print0 | xargs -0 cat | grep st2actions ; test $$? -eq 1
+	find ${ROOT_DIR}/st2common/st2common/ \( -name \*.py ! -name runnersregistrar\.py -name \*.py ! -name compat\.py ! -name inquiry\.py \) -type f -print0 | xargs -0 cat | grep st2actions ; test $$? -eq 1
```

Comment:
@m4dcoder Is this change acceptable to you? I think this is what you were trying to do with [this commit](https://github.com/StackStorm/st2/pull/4309/commits/e88323eaf4c6bf013c3139cdc09d82b7f4706dac#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R268), but I want to double check with you.

Without this fix we get two errors: `find: missing )` and `Command not found '-name'`, which I think is caused by the `|` character terminating the `find` command prematurely.

I also noticed that `-name \*.py` is in the `find` parameters twice, but I don't think that would cause any issues.